avalon/harmony/lib.py

```diff
@@ -55,7 +56,33 @@ def launch(application_path):
     self.application_path = application_path
 
     # Launch Harmony.
-    os.environ["TOONBOOM_GLOBAL_SCRIPT_LOCATION"] = os.path.dirname(__file__)
+    if os.getenv("TOONBOOM_GLOBAL_SCRIPT_LOCATION"):
+
+        avalon_harmony_startup_path = \
+            os.path.join(os.getenv("TOONBOOM_GLOBAL_SCRIPT_LOCATION"),
+                         "TB_sceneOpened.js")
+
+        env_harmony_startup_path = \
+            os.path.join(os.path.dirname(__file__), "TB_sceneOpened.js")
+
+        if not filecmp.cmp(avalon_harmony_startup_path,
```

Comment:
What is `filecmp`?

Reply:
Simple non-expensive byte comparison of 2 files. 
See: https://stackoverflow.com/questions/1072569/see-if-two-files-have-the-same-content-in-python

Reply:
Dont see any import?

Reply:
Sorry, ignore that